ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -4,15 +4,12 @@
 import os
 import re
 
-from ner_v2.detectors.numeral.constant import NUMBER_RANGE_KEYWORD_FILE_NAME, NUMBER_RANGE_FILE_VARIANTS_COLUMN_NAME, \
-    NUMBER_RANGE_FILE_POSITION_COLUMN_NAME, NUMBER_RANGE_FILE_RANGE_TYPE_COLUMN_NAME, NUMBER_RANGE_MIN_TYPE, \
-    NUMBER_RANGE_MAX_TYPE, NUMBER_RANGE_MIN_MAX_TYPE, NUMBER_REPLACE_TEXT, NUMBER_RANGE_MIN_VALUE, \
-    NUMBER_DETECTION_RETURN_DICT_VALUE, NUMBER_RANGE_MAX_VALUE, NUMBER_RANGE_VALUE_UNIT, \
-    NUMBER_DETECTION_RETURN_DICT_UNIT
+import ner_v2.detectors.numeral.constant as numeral_constant
 from ner_v2.detectors.numeral.utils import get_list_from_pipe_sep_string
 from ner_v2.detectors.numeral.number.number_detection import NumberDetector
 
 NumberRangeVariant = collections.namedtuple('NumberRangeVariant', ['position', 'range_type'])
+NumberTaggedMap = collections.namedtuple('NumberTaggedMap', ['entity_value', 'original_text'])
```

Comment:
Slightly confusing name as Map implies it might be a dict. Something along the lines of `ValueTextPair` might be more explicit

Reply:
Done